plugins/woocommerce-admin/client/order-attribution-install-banner/header-banner/index.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import { Button } from '@wordpress/components';
+import { useEffect } from '@wordpress/element';
+import { plugins } from '@wordpress/icons';
+import { __ } from '@wordpress/i18n';
+import { recordEvent } from '@woocommerce/tracks';
+import { getPath } from '@woocommerce/navigation';
+
+/**
+ * Internal dependencies
+ */
+import { useOrderAttributionInstallBanner } from '~/order-attribution-install-banner';
+import './style.scss';
+
+export const HeaderBanner = () => {
```

Comment:
Now that we created this banner I have my thoughts about the folder and file naming. 

How many banners do we plan to have under the `order-attributtion-install-banner` folder?  

As is now, I would advocate for renaming the other banner  as "MainBanner" and place it in its own folder `main-banner` also I believe the folder can be `order-attributtion-install-banners` now. 

So:

- order-attributtion-install-banners 
   - header-banner (folder)
   - main-banner (folder)
   - index.js
   - use-order-attribution-install-banner.js 

order-attribution-install-banners

Reply:
However, and if there is only 2 banner we can consider to use the order-attribution-install-banner adding a prop isHeaderBanner instead of creating a new component? So we can re-use some logic:

```jsx
/**
 * External dependencies
 */
import { Button, Card, CardBody } from '@wordpress/components';
import { plugins } from '@wordpress/icons';
import { useEffect } from '@wordpress/element';
import { __ } from '@wordpress/i18n';
import { Text } from '@woocommerce/experimental';
import { recordEvent } from '@woocommerce/tracks';
import { getPath } from '@woocommerce/navigation';

/**
 * Internal dependencies
 */
import OrderAttributionInstallBannerImage from './order-attribution-install-banner-image';
import { useOrderAttributionInstallBanner } from './use-order-attribution-install-banner';
import './style.scss';

export const OrderAttributionInstallBanner = ( {
	bannerImage = <OrderAttributionInstallBannerImage />,
	eventContext = 'analytics-overview',
	isHeaderBanner = false,
} ) => {
	const { isDismissed, dismiss, shouldShowBanner } =
		useOrderAttributionInstallBanner();

	const onClick = () => {
		recordEvent( 'order_attribution_install_banner_clicked', {
			path: getPath(),
			context: eventContext,
		} );
	};

	const link = 'https://woocommerce.com/products/woocommerce-analytics';

	const shouldRender =
		( ! isHeaderBanner && ( shouldShowBanner && ! isDismissed ) ) ||
		( isHeaderBanner && ( shouldShowBanner && isDismissed ) );
	
	useEffect( () => {
		if ( ! shouldRender ) {
			return;
		}

		recordEvent( 'order_attribution_install_banner_viewed', {
			path: getPath(),
			context: eventContext,
		} );
	}, [ eventContext, shouldRender ] );

	if ( ! shouldRender ) {
		return null;
	}

	if ( isHeaderBanner ) {
		return (
			<Button
				className="woocommerce-order-attribution-install-header-banner"
				href={ link }
				variant="secondary"
				icon={ plugins }
				size="default"
				onClick={ onClick }
			>
				{ __( 'Try Order Attribution', 'woocommerce' ) }
			</Button>
		);
	}

	return (
		<Card
			size="medium"
			className="woocommerce-order-attribution-install-banner"
		>
			<CardBody className="woocommerce-order-attribution-install-banner__body">
				<div className="woocommerce-order-attribution-install-banner__image_container">
					{ bannerImage }
				</div>
				<div className="woocommerce-order-attribution-install-banner__text_container">
					<div className="woocommerce-order-attribution-install-banner__text-badge">
						<Text
							className="woocommerce-order-attribution-install-banner__text-description"
							as="p"
							size="12"
							align="center"
						>
							{ __( 'New', 'woocommerce' ) }
						</Text>
					</div>
					<Text
						className="woocommerce-order-attribution-install-banner__text-title"
						as="p"
						size="16"
					>
						{ __(
							'Discover what drives your sales',
							'woocommerce'
						) }
					</Text>
					<Text
						className="woocommerce-order-attribution-install-banner__text-description"
						as="p"
						size="12"
					>
						{ __(
							'Understand what truly drives revenue with our powerful order attribution extension. Use it to track your sales journey, identify your most effective marketing channels, and optimize your sales strategy.',
							'woocommerce'
						) }
					</Text>
					<div>
						<Button
							href={ link }
							variant="primary"
							onClick={ onClick }
						>
							{ __( 'Try it now', 'woocommerce' ) }
						</Button>
						<Button
							variant="tertiary"
							onClick={ () => dismiss( eventContext ) }
						>
							{ __( 'Dismiss', 'woocommerce' ) }
						</Button>
					</div>
				</div>
			</CardBody>
		</Card>
	);
};

```


```jsx
{ isAnalyticsOverviewScreen && (
					<OrderAttributionInstallBanner
						isHeaderBanner={ true }
						eventContext="analytics-overview-header"
					/>
				) }
```